src/app/core/entity-components/previous-relations/time-period-relation.ts

```diff
@@ -0,0 +1,61 @@
+import { Entity } from "../../entity/model/entity";
+import { DatabaseEntity } from "../../entity/database-entity.decorator";
+import { DatabaseField } from "../../entity/database-field.decorator";
+import moment from "moment";
+
+/**
+ * Record that is only active for a given time period.
+ */
+@DatabaseEntity("TimePeriodRelation")
+export class TimePeriodRelation extends Entity {
```

Comment:
Why does it have `Relation` in the name? There is no hard-coded property that relates it to any other entity.

Reply:
True. Same as the component, the name should be improved and both names (entity + component) aligned.